tests/transition_to_if.py

```diff
@@ -63,7 +64,9 @@
     assert cluster.biosNode.waitForLibToAdvance(), "Lib should advance after instant finality activated"
     assert cluster.biosNode.waitForProducer("defproducera"), "Did not see defproducera"
     assert cluster.biosNode.waitForHeadToAdvance(blocksToAdvance=13) # into next producer
```

Comment:
Not from your PR. You can add an error message like head not reached 13 on biosNode.

Reply:
Done